src/zarr/array.py

```diff
@@ -343,6 +343,16 @@ def ndim(self) -> int:
     def shape(self) -> ChunkCoords:
         return self.metadata.shape
 
+    @property
+    def chunks(self) -> ChunkCoords:
+        try:
+            chunk_shape = getattr(self.metadata.chunk_grid, "chunk_shape")  # noqa: B009
+            return chunk_shape
```

Comment:
```suggestion
            return self.metadata.chunk_grid.chunk_shape
```

B009 is actually useful here. 

> B009: Do not call getattr(x, 'attr'), instead use normal property access: x.attr. Missing a default to getattr will cause an AttributeError to be raised for non-existent properties. There is no additional safety in using getattr if you know the attribute name ahead of time.

Reply:
Ok done.